benefits/admin.py

```diff
@@ -0,0 +1,36 @@
+from django.contrib import admin
+from django.contrib.auth.models import Group
+from django.template.response import TemplateResponse
+from benefits.core.models import TransitAgency
+from benefits.core import session
+
+
+STAFF_GROUP_NAME = "Cal-ITP"
+
+
+class BenefitsAdminSite(admin.AdminSite):
+
+    def index(self, request, extra_context=None):
+        """
+        Display the main admin index page, which lists all of the installed
+        apps that have been registered in this site.
+        """
```

Comment:
I appreciate these comments!

Reply:
They came from Django but I agree that they are helpful! 🙂

Reply:
Yep, updated the docstring in 772f369554c6e8eded3cb4eb65913ed93ba9e8a5